synapse/storage/events.py

```diff
@@ -193,7 +193,9 @@ def f(self, *args, **kwargs):
     return f
 
 
-class EventsStore(EventsWorkerStore):
+# inherits from EventFederationStore so that we can call _update_backward_extremities
+# and _handle_mult_prev_events (though arguably those could both be moved in here)
+class EventsStore(EventFederationStore):
```

Comment:
Do we want to remove the explicit include of `EventsWorkerStore` and rely on that `EventFederationStore` pulls in it? Personally I'd prefer to keep it.

Reply:
don't mind really. I guess it makes sense to keep it explicit so we don't forget it's needed.